### PR TITLE
dfgdfg

### PR DESCRIPTION
**Pull request by way0utwest**

### CODE REVIEW

**Review by BecaLParker — COMMENTED** [review]
Looks good to me. Unless @ScaryDBA has a problem with this?

**Comment by ScaryDBA** [comment]
Column names and data types are wrong, wrong, wrong.

**Comment by way0utwest** [comment]
fine, fixed it

File: DBCode/16_MissingView.sql [hunk 1]
```diff
@@ -17,11 +17,17 @@ GO
...
-SELECT TOP 10 
-       *
+SELECT TOP(10)
+ e.EventName,
+ e.EventDate,
+ e.EventLocation,
+ e.CountryCode        
 FROM dbo.Event e
+INNER JOIN dbo.CountryCodes c ON c.CountryCode = e.CountryCode 
 ORDER BY EventDate DESC;
```
Comment (BecaLParker) [inline DBCode/16_MissingView.sql:30]: I can add a review annoation to a specific line

**Pull request merged**